discovery-provider/src/solana/anchor_program_indexer.py

```diff
@@ -126,6 +126,15 @@ def process_transactions(
         records: List[Any] = []
         for transaction in processed_transactions:
             instructions = transaction["tx_metadata"]["instructions"]
+            meta = transaction["result"]["meta"]
+            error = meta["err"]
+
+            if error:
```

Comment:
what do you think of moving this to be a part of is_valid_instruction?

Reply:
https://linear.app/audius/issue/PLAT-128/minor-follow-ups-to-indexing-admin-user-functions-pr